### ISSUE TITLE
[MILESTONE CHECKLIST] 3. Migrate and first-pass polish all interactive content

### ISSUE THREAD

**Issue by NatalieZelenka** [NatalieZelenka/phenotype_from_genotype#26]
Migrate all interactive content (jupyter notebooks) for the following chapters (in this order):
- [x] Chapter 6: Filter
   - [x] CAFA3 version
   - [x] Discussion
   - [x] Future work
- [x] Chapter 3: Compbio background + reproducibility
- [x] Chapter 4: Snowflake
    - [x] Plan chapter in detail (#40, )
    - [x] Draft chapter (#41,)

This includes:
* Migrating the actual code (non-code sections should be finished as part of milestone 1.)
* Finishing the research for each of these chapters. Each notebook should be self-contained and finished.
* Finalising a sensible jupyter book sidenav (config) layout, and the layout/story for each chapter.
* Finalising the text in these chapters (for example, rewriting the compio background chapter, which has a lot of non-interactive content)
* Citations/Figures/Cross-refs
* TODOs for discussion/intro/abstract

This milestone does not include:
* Adding tests or other infrastructure for the interactive content, except for any that absolutely MUST be written to migrate/understand it.
* Adding additional research (extra notebooks can be added later if they are absolutely needed).
* Migrating chapters 1, 2, and 7 (they don't include any code), or chapter 5 (should be done before this milestone (#25))
* Nice-to-haves like hand-drawn illustrations, asides, epigraphs, etc.

## Decided not to do (for now)
- filip
    - Release and writeup as a tool #35 
    - Include combined dataset in filip
-  Modelling study of genes
    - Distribution of publications over genes (#22, ) & How does this affect Filp?
     - How Complete is the gene ontology? (#23, ) & How does this affect Filp?
-  MAPS (#24, ) & how does this affect Computational Biology? I.e. do a multiverse of a comp bio question?
